shaas.go

```diff
@@ -88,6 +90,28 @@ func handleExit(res http.ResponseWriter, req *http.Request) {
 	os.Exit(code)
 }
 
+func handleHealth(res http.ResponseWriter, req *http.Request) {
+	responseCode := 200
+	var err error
+	if c := req.URL.Query().Get("responseCode"); c != "" {
+		if responseCode, err = strconv.Atoi(c); err != nil {
+			responseCode = 200
+		}
+	}
+
+	delayMilliSecs := 10
```

Comment:
would `0` be a better default?

Reply:
sure, was just an arbitrary small number